README.md

```diff
@@ -1,17 +1,17 @@
 Numix icon theme
 ================
-Numix is the official icon theme from the [Numix Project](http://numixproject.org). It is heavily inspired by, and based upon parts of the Elementary, Humanity and Gnome icon themes. Designed to be used along side the application icon theme [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle). This readme provides information on [installation](https://github.com/numixproject/numix-icon-theme#installation) and [icon requests](https://github.com/numixproject/numix-icon-theme#icon-requests). Licensed under the GPL-3.0+
+Numix is the official icon theme from the [Numix Project](http://numixproject.org). It is heavily inspired by, and based upon parts of the Elementary, Humanity and Gnome icon themes. Numix is designed to be used along-side an application icon theme like [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle) or [Numix Square](https://github.com/numixproject/numix-icon-theme-square). This readme provides information on [installation](https://github.com/numixproject/numix-icon-theme#install-it) and [icon requests](https://github.com/numixproject/numix-icon-theme#icon-requests). Licensed under the GPL-3.0+
+
+## Install it
 
-### Installation
 If using this with one of our app icon themes (like [Numix Circle](https://github.com/numixproject/numix-icon-theme-circle)) make sure you install both parts using the same method. This makes sure that the panel icons keep working as intended.
 
-If you use Ubuntu or any of its derivatives (including Mint and elementary OS) you can use our Numix PPA. Fire up a Terminal and run the following:
+### Official Distro Packages
+|Distro|Install Command/Links|
+|:----:|:----:|
+|[![debian](https://www.debian.org/logos/openlogo-nd-25.jpg)](https://www.debian.org/)&nbsp;![ubuntu](https://dl.dropboxusercontent.com/u/60521097/ubuntu_orange_hex.png)|`apt install numix-icon-theme`|
```

Comment:
Doing the list of available packages as a table rather than in paragraphs is a neat idea, but if you're going to do it you need to commit to it. Having only one package listed there kinda defeats the point. To merge the "unofficial" and "official" sections maybe the format below could be used?

Distro | Repo | Release | Install Commands
--- | --- | --- | --- 
Debian/Ubuntu | Universe | stable | <code>sudo apt install numix-icon-theme</code>
Ubuntu | PPA | weekly | `sudo add-apt-repository ppa:numix/ppa` <br> `sudo apt-get update` <br> `sudo apt-get install numix-icon-theme`
Fedora | Copr | stable | `sudo dnf copr enable numix/numix` <br> `sudo dnf install numix-icon-theme`
Arch | AUR | git | `sudo yaourt numix-icon-theme-git`